charts/karavi-observability/templates/cert-manager.yaml

```diff
@@ -91,45 +93,41 @@ spec:
   - otel-collector
   - otel-collector.karavi.svc.kubernetes.local
```

Comment:
Shouldn't we variable-ize the dns name with the namespace ?

If I install the module in `powerflex` NS the service URL will be `otel-collector.powerflex.svc.kubernetes.local`